dev.mrquantumoff.mcmodpackmanager.metainfo.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.mrquantumoff.mcmodpackmanager</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+  <name>Minecraft Modpack Manager</name>
+  <summary>Manage your modpacks with ease!</summary>
+  <icon type="remote" width="256" height="256">
```

Comment:
please don't use remote icons.

Reply:
is using stock icons ok?

Reply:
So, which type of icons should I use? If I have to use "cached" or "local" will it be correctly displayed with GNOME Software/Discover?

Reply:
just remove the icon entry.

Reply:
Ok